cmd/limactl/shell.go

```diff
@@ -78,6 +55,34 @@ func shellAction(clicontext *cli.Context) error {
 		return err
 	}
 
+	// Buildup changeDirCmd to "cd workDir || cd currentDir || cd homeDir"
+	var changeDirCmd string
+	workDir := clicontext.String("workdir")
+	if workDir == "" {
+		changeDirCmd = "false"
```

Comment:
What is this?

Reply:
How can `currentDir` (from the host) work when there are no mounts? I mean, potentially you could be in `/tmp` and then change to `/tmp` inside the guest, but when does that ever make sense? I think when you don't specify a `--workdir` with a VM that has no mounts, you should end up in the home directory inside the VM.

So `false` is just something that returns a non-zero exit code, because when there are mounts, the command becomes:

```
false || cd currentDir || cd homeDir
```

We must have something on the left side of the `||` that fails (but doesn't show an error on stderr), so the `cd currentDir` will be tried next.

Reply:
Thanks for explanation. After merging this, I'll add logrus.Debug for better readability